dagfactory/dagbuilder.py

```diff
@@ -396,21 +402,27 @@ def build(self) -> Dict[str, Union[str, DAG]]:
                     task_conf["task_group"] = task_groups_dict[
                         task_conf.get("task_group_name")
                     ]
+                params: Dict[str, Any] = {
```

Comment:
I think you should move back this `if` statement one level up, so you can pass the params to the `generator`. It will need the params to know what to generate!